service-graph/design.md

```diff
@@ -1,5 +1,22 @@
 # Service Graph
 
+## Graph Zoom Tool
+The graph zoom tool contains buttons that enable the user to zoom the graph in and out as well as fit the zoom to the current view. This tool may also contain a trigger for the graph legend as well as a graph drawing algorithm selector.
```

Comment:
should "also contain" be "optionally contain"?

Reply:
and we should standardize to use either "graph drawing algorithm" or "graph format"